scripts/us_census/acs5yr/subject_tables/subject_table_test.py

```diff
@@ -133,6 +141,6 @@ def test_csv_mcf_column_map(self):
                     self.assertEqual(test_cmap, expected_cmap)
 
                     # Test CSV
-                    test_csv, expected_csv = _read_files(
-                        test_csv_path, paths['csv'])
-                    self.assertEqual(test_csv, expected_csv)
+                    csv_compare_bool = _test_csv_files(test_csv_path,
+                                                       paths['csv'])
+                    self.assertEqual(csv_compare_bool, True)
```

Comment:
self.assertTrue(csv_compare_bool)

Reply:
Changed the assertion to `assert_frame_equal(test_df, expected_df)`.